src/applications/personalization/dashboard-2/components/Dashboard.jsx

```diff
@@ -138,7 +141,7 @@ const mapStateToProps = state => {
   // available. And we will need to show the health care section if they are a
   // patient and/or have rx or msg services available
   const showClaimsAndAppeals = isLOA3;
-  const showHealthCare = isLOA3;
+  const showHealthCare = isLOA3 && isVAPatient;
```

Comment:
Do not show the health care section when someone is NOT enrolled in VA health care